web/pgadmin/static/js/components/ReactCodeMirror/components/Editor.jsx

```diff
@@ -193,6 +193,12 @@ export default function Editor({
             }
             if(update.docChanged) {
               onChange?.(update.view.getValue(), update.view);
+              // Update editable properties
+              editor.current?.dispatch({
```

Comment:
So you're dispatching an event on every doc change. Not at all good for performance.

Reply:
As i try to dispatch that event from another component but not worked.
Will check once again.

Reply:
I have changed the approach as per your suggestions.
Thanks